src/function/table/free_space_info.cpp

```diff
@@ -23,9 +23,10 @@ struct FreeSpaceInfoBindData final : TableFuncBindData {
 static common::offset_t internalTableFunc(const TableFuncMorsel& morsel,
     const TableFuncInput& input, common::DataChunk& output) {
     const auto bindData = input.bindData->constPtrCast<FreeSpaceInfoBindData>();
-    const auto entries =
-        bindData->ctx->getStorageManager()->getDataFH()->getPageManager()->getFreeEntries(
-            morsel.startOffset, morsel.endOffset);
+    const auto entries = storage::StorageManager::Get(*bindData->ctx)
```

Comment:
Can you wrap this to `PageManager::Get(*bindData->ctx)`?

Reply:
Done with https://github.com/kuzudb/kuzu/pull/5909/commits/54c7691abfa546a921514e399ae08221e3c4cf5c